landoapi/bmo.py

```diff
@@ -8,8 +8,8 @@
 
 
 def bmo_bug_endpoint() -> str:
```

Comment:
Should rename this to `bmo_uplift_endpoint` or something else less general, since there is nothing in the module name or method name hinting that this is an uplift related method.